src/assessment.js

```diff
@@ -50,4 +50,5 @@ var q2 = [/* your answer here */];
 //
 var q5 = function fireOnAlderaan () {
   // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
same problem of understanding the question correctly. I noticed the words "bind it (or, change `this`) to `alderaan` before executing it" in the question, that's why i didn't do deathStar.fire.bind(alderaan)(); I actually tried the latter in repl.it and it works, but because of the "before executing it" thing, i didn't put that as my answer.